editor/src/components/inspector/inspector-common.ts

```diff
@@ -557,3 +557,27 @@ export function addPositionAbsoluteTopLeft(
     setProperty('always', elementPath, styleP('position'), 'absolute'),
   ]
 }
+
+export function notFixedSizeOnEitherAxis(
```

Comment:
Minor, but please can you rename this to `notFixedSizeOnBothAxes`. The current name implies that we're checking `horizontalState !== 'fixed' || verticalState !== 'fixed'`

Reply:
Absolutely! fixed in https://github.com/concrete-utopia/utopia/pull/3215/commits/8a22e1289b1d3054f2c8d3f847c0e50f1a5ca6ca